src/main/resources/assets/additionaladditions/lang/vi_vn.json

```diff
@@ -0,0 +1,88 @@
+  "enchantment.additionaladditions.precision.desc": "Cải thiện độ chính xác của nỏ.",
+  "enchantment.additionaladditions.speed": "Tốc độ",
+  "enchantment.additionaladditions.speed.desc": "Tăng tốc độ di chuyển.", 
+  "entity.additionaladditions.glow_stick": "Gậy phát sáng",
+
+  "depth_meter.elevation": "Độ cao: %s",
+
+  "item.minecraft.potion.effect.haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.potion.effect.long_haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.potion.effect.strong_haste_potion": "Thuốc của sự vội vàng",
+  "item.minecraft.splash_potion.effect.haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.splash_potion.effect.long_haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.splash_potion.effect.strong_haste_potion": "Thuốc của sự vội vàng có thể ném được",
+  "item.minecraft.lingering_potion.effect.haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.lingering_potion.effect.long_haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.lingering_potion.effect.strong_haste_potion": "Thuốc của sự vội vàng kéo dài",
+  "item.minecraft.tipped_arrow.effect.haste_potion": "Mũi tên vội vàng",
+  "item.minecraft.tipped_arrow.effect.long_haste_potion": "Mũi tên vội vàng",
+  "item.minecraft.tipped_arrow.effect.strong_haste_potion": "Mũi tên vội vàng",
+
+  "additionaladditions.advancements.obtain_adoghr_disc": "adoghr",
+  "additionaladditions.advancements.obtain_adoghr_disc.desc": "Sở hữu một trong những đĩa nhạc của adoghr",
+  "additionaladditions.advancements.obtain_chicken_nugget": "What a Nugget, Once Again!",
+  "additionaladditions.advancements.obtain_chicken_nugget.desc": "Find a Chicken Nugget in the wilderness",
+  "additionaladditions.advancements.obtain_gilded_netherite": "Mạ vàng",
+  "additionaladditions.advancements.obtain_gilded_netherite.desc": "Sở hữu áo giáp Netherit mạ vàng",
+  "additionaladditions.advancements.obtain_glow_stick": "Sáng lên đi em ei!",
+  "additionaladditions.advancements.obtain_glow_stick.desc": "Dùng Gậy phát sáng để thắp sáng hang động",
+  "additionaladditions.advancements.obtain_mysterious_bundle": "Nhắm mắt mua liều",
+  "additionaladditions.advancements.obtain_mysterious_bundle.desc": "Mua Túi bọc bí ẩn từ một thương nhân lang thang",
+  "additionaladditions.advancements.obtain_pocket_jukebox": "What a Nugget!",
+  "additionaladditions.advancements.obtain_pocket_jukebox.desc": "Craft the Pocket Jukebox to listen to music on the go",
+  "additionaladditions.advancements.obtain_rose_gold": "Vàng và Hoa hồng",
+  "additionaladditions.advancements.obtain_rose_gold.desc": "Sở hữu áo giáp bằng vàng hồng",
+  "additionaladditions.advancements.obtain_scoped_crossbow": "Sát thủ",
+  "additionaladditions.advancements.obtain_scoped_crossbow.desc": "Sở hữu Nỏ với ống ngắm",
+  "additionaladditions.advancements.shot_self_scoped_crossbow": "Sát thủ tự hủy",
```

Comment:
Would be nice if you let me know the meaning of this translation

Reply:
"tự hủy" or "tự bóp", yes it's self destruct, we use those words as a trend here